e2e/scripts/st_arrow_dataframe_unevaluated_snowpark_dataframe.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
```

Comment:
And can you rename this file to `st_arrow_unevaluated_snowpark_dataframe.py` and also test this for `line_chart`, `bar_chart`, `area_chart` and `table`

Reply:
It's renamed now and almost all the tests you mentioned are present, ⚠️ with the exception:

* st.table renders properly, however it takes so looooong the E2E tests for every PR already are taking much time, I've decided to left it out in this test, is that ok?

Reply:
yep, that's fine. Thought `st.table` might need some investigation on why it is so slow. I think it would speed it up if we use fewer columns (e.g. just 5)

Reply:
btw. what does slow mean, how long does it run (with & without `st.table`?)

Reply:
Sure 👍 It's lowered to 4 now 👍